File: project/.sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ProjectNameSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ProjectNameSDK.test()
    // console.log('testsdk', testsdk)
    equal(null !== testsdk, true)
  })

})

File: ts/project/.sdk/tm/swift/Tests/ProjectNameSDKTests/ExistsTest.swift
// ProjectName SDK exists test.

import XCTest

@testable import ProjectNameSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = ProjectNameSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}
